### PR TITLE
Workbench on gcp

### PR DESCRIPTION
**Pull request by DmytroYesin**

### CODE REVIEW

**Comment by naamaesa** [comment]
what is the issue here?

File: source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+**Requirements:**
+
+---
+- <span>*</span> Experience with Google Cloud Platform.
+- <span>*</span> Google Cloud Project.  
+         
+<span></span>        
+              
+##Create Virtual Machine
```
Comment (gfedorenko) [inline source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown]: Add whitespace after ## `## Create Virtual Machine`

File: source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+* <span>*</span> On the left sidebar find **Compute Engine -> VM instances.**
+* <span>*</span> At the top of the page click **CREATE INSTANCE**.
+* <span>*</span> Change name of the instance from _instance-1_ to a better name.
+* <span>*</span> Choose **region** and **zone**, that fits your needs best.
+* <span>*</span> Select machine type, recommended is **n1-standart-8**, required is **n1-standart-4**.
+* <span>*</span> Increase size of the boot disk, recommended size is **60GB**, required is **45GB**.
+* <span>*</span> Don't change default OS image: _Debian GNU/Linux 9 (stretch)_.
+* <span>*</span> Check two checkboxes under the **Firewall** section.
+
+>  <pre>  <input type="checkbox" id disabled class="task-list-item-checkbox" checked> Allow HTTP traffi
```
Comment (gfedorenko) [inline source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown]: traffic

File: source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+* <span>*</span> **Double check** your settings before creating.
+  
+<img class="col-md-12" src="https://camo.githubusercontent.com/087e942149862f48dc0177d051842371e62a2254/68747470733a2f2f64726976652e676f6f676c652e636f6d2f75633f69643d314c536c746d4944744c714b786d6b435a7030324f485a654d375f637630595456" alt="Settings before VM Creation">
+   
+* <span>*</span> Click **Create** button at the bottom of the page.    
+       
+<span></span>
+
+##Post creation steps
```
Comment (gfedorenko) [inline source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown]: Add whitespace

File: source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+<span></span>
+
+<p>
+  <img class="col-md-12" src="https://camo.githubusercontent.com/2c1b9239396fd62558acacabe60c0f90120234b1/68747470733a2f2f64726976652e676f6f676c652e636f6d2f75633f69643d316c6a52786f68756a396c326c786d5a6d6d495a516d36635a4356344661376958" alt="Settings before VM Creation">
+</p>
+
+<span></span>
+
+##Install workbench
```
Comment (gfedorenko) [inline source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown]: Add whitespace

File: source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown [hunk 5]
```diff
@@ -8,183 +8,152 @@ tags: [ Rubykube, Tutorials ]
...
-
-<h2 class="mt-4 mb-3">Post creation steps</h2>
-<p>Here is your created vm. In the green rectangle you will find public IP of your virtual machine.</p>
+**Requirements:**
+
+---
+- <span>*</span> Experience with Google Cloud Platform.
+- <span>*</span> Google Cloud Project.  
+         
+<span></span>
```
Comment (gfedorenko) [inline source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown]: Do we need this empty span?
There are a few more later in doc

Comment (DmytroYesin) [reply source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown]: They are for increasing the padding between blocks of text

Comment (gfedorenko) [reply source/blog/articles/Tutorials/2018-10-01-workbench_on_gcp.html.markdown]: @DmytroYesin lets just put <br/> tags

**Pull request merged**